debugtools/DDR_VM/src/com/ibm/j9ddr/CompatibilityConstants29.dat

```diff
@@ -76,3 +76,8 @@ MM_ForwardedHeader.OMR_FORWARDED_TAG_MASK = 0
 ShcdatatypesConstants.TYPE_PREREQ_CACHE = -1
 
 ShCFlags.J9SHR_DATA_TYPE_STARTUP_HINTS = -1
+
+J9ArrayShapeFlags.J9ArrayShape8Bit = 0
+J9ArrayShapeFlags.J9ArrayShape16Bit = 1
+J9ArrayShapeFlags.J9ArrayShape32Bit = 2
+J9ArrayShapeFlags.J9ArrayShape64Bit = 3
```

Comment:
I don't see any value in adding these constants here. We can wait until they're actually needed by DDR.

Reply:
Sounds good, fixed in f125181